### PR TITLE
ui.json schema validation added

### PR DESCRIPTION
**Pull request by zachkirlewmrf**

### CODE REVIEW

File: json-lint/test/resources/schema/ui/valid/ui.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "fontWeight" : "NORMAL"
+        },
+        "body" : {
+            "fontFamily" : "HelveticaNeue",
+            "fontSize" : "MEDIUM",
+            "fontWeight" : "NORMAL"
+            }
+        },
+        "mainLogoRatio" : 0,
+        "SomeNewProperty" : "Look mum! I can code!"
```
Comment (gianmarcobrunialtimrf) [inline json-lint/test/resources/schema/ui/valid/ui.json]: On a JSON template? Yeah right.

Comment (zachkirlewmrf) [reply json-lint/test/resources/schema/ui/valid/ui.json]: Loooooooool

**Comment by gianmarcobrunialtimrf** [comment]
Good to go for me. Manage the deploy with @eduardcotmrf in order for all the teams to be aware of these new validations.

**Comment by zachkirlewmrf** [comment]
retest this please

**Pull request merged**